### ISSUE TITLE
Further Improvements

### ISSUE THREAD

**Issue by NoHandsMate** [carminecolucci/HiDDeN-keras#3]
The implementation uses as noise layers only 
1. Identity
2. Gaussian (2)
3. Dropout (30%)

Maybe we can add at least JPEG and Crop. Keras implements ```keras.layers.CenterCrop```.
